optimism/src/keccak/environment.rs

```diff
@@ -63,23 +63,31 @@ pub struct KeccakEnv<F> {
     pad_suffixes: [[F; PAD_SUFFIX_LEN]; RATE_IN_BYTES],
 }
 
-impl<F: Field> KeccakEnv<F> {
-    /// Starts a new Keccak environment for a given hash index and bytestring of preimage data
-    pub fn new(hash_idx: u64, preimage: &[u8]) -> Self {
```

Comment:
Was `preimage` unused before?

Reply:
Not really, it is used from the witness env of the mips side to create a new keccak env after finding a syscall instruction.